Apps/WebClient/src/ClientApp/app/views/landing.vue

```diff
@@ -164,8 +164,6 @@
           <div class="title-text">
             <div>
               Empowering you
-            </div>

```

Comment:
This was necessary to make the sentence break as per the design.

Reply:
The landing needs more work. I will update all and make the necessary changes.